src/main/java/com/backbase/ct/bbfuel/configurator/TransactionsConfigurator.java

```diff
@@ -54,4 +55,24 @@ public void ingestTransactionsByArrangement(String externalArrangementId, boolea
 
         log.info("Transactions [{}] ingested for arrangement [{}]", randomAmount, externalArrangementId);
     }
+
+    /**
+     * Ingest transactions that will be referenced to the specified pockets.
+     * @param externalArrangementId the parent pocket external arrangement id
+     * @param ingestedPockets list of ingested pockets
+     */
+    public void ingestTransactionsForPocket(String externalArrangementId, List<Pocket> ingestedPockets) {
+        List<TransactionsPostRequestBody> transactions = Collections.synchronizedList(new ArrayList<>());
```

Comment:
This list doesn't need to be synchronized either, I wonder why you've done it :thinking:

Reply:
changed